### DISCUSSION TITLE
SONOFF DUAL R3 Lite come deviatore

### DISCUSSION THREAD

**Discussion by diaz66** [indomus/forum#3208] [Hardware & Firmware]
Un saluto a tutta la Community.

E possibile ottenere quanto in oggetto ?

e cioè da così 
![image](https://github.com/user-attachments/assets/01c81b73-e027-470c-bd76-3aa1a23cfdff)

a così
![image](https://github.com/user-attachments/assets/f3fb532a-e777-453c-8897-2e038714b5b8)

Non mi è mai capitato di utilizzarlo così ma ho necessità. Penso non ci siano problemi. Voi che dite ?

**Comment by indomus** [comment]
Non ti avventurare. Sebbene lo schema che hai postato potrebbe avere senso, consultati con un elettricista capace.

**Reply by diaz66** [reply]
Grazie per la risposta. Non sono un elettricista ma conosco comunque la materia e penso che non ci siano errori. Quando mi arriva proverò e farò sapere. 

Casomai , il problema , potrebbe esserci quando si comanda il sonoff tramite app e/o Home Assistant. In questo caso, bisognerà fare in modo che ci sia sempre e solo una uscita attiva verso il secondo deviatore. Ci dovrebbe essere la funzione di interlook che evita proprio che entrambe le uscite siano in fase (per esempio quando si utilizza per comandare motori su/giù tapparelle).
